openupgrade_scripts/scripts/mail/15.0.1.5/post-migration.py

```diff
@@ -1,5 +1,11 @@
+import html
+import re
+
+from bs4 import BeautifulSoup
```

Comment:
Is this library included in standard Odoo?

Reply:
I don't think so. Is there a mechanism to declare third party library dependencies?

Reply:
That's a problem, as doing the upgrade, this will fail. Can you do the transformations without requiring this library?

Reply:
> I don't think so. Is there a mechanism to declare third party library dependencies?

I'm working here #3854 to use classic generation of requirements.txt based on the dependency (still wip). If it's ok, we can just add BeautifulSoup as a dependency of openupgrade_scripts module in the manifest.
Then just change the doc : "pip install openupgradelib" by "pip install -r ./requirements.txt".
That should do the job. Don't you think ?